src/operator-manual/metrics/02-reference.md

```diff
@@ -0,0 +1,31 @@
+# Metrics Reference
+
+Kubewarden exposes some relevant metrics that enhaces visibility of the platform, and allows cluster
+administrators and policy developers to identify patterns and potential issues.
+
+## Policy Server
+
+The Policy Server is the component that initializes and runs policies. Upon receiving requests from
+the Kubernetes API server, it will forward the request to the policy, and return the response
+provided by the policy to the Kubernetes API server.
+
+### Metrics
+
+Name | Type | |
+--- | --- | ---
+`kubewarden_policy_evaluations_total` | Counter | [Baggage](#kubewarden_policy_evaluations_total)
+
+#### `kubewarden_policy_evaluations_total`
+
+##### Baggage
+
```

Comment:
Maybe add two lines explaining what a baggage is.

Just from the top of my head:

> Baggage are key-value attributes added to the metric. The baggage keys can be used to further refine the metrics.

Reply:
I'm adding it to the section before because the Baggage subsection is going to be repeated for every metric.